WebService/v1/Controllers/SimulationsController.cs

```diff
@@ -95,19 +100,31 @@ public async Task<SimulationApiModel> GetAsync(string id)
                 simulation, this.servicesConfig, this.deploymentConfig, this.connectionStringManager, this.simulationRunner, this.rateReporter);
         }
 
+        [HttpPost("{id}/metrics/iothub!search")]
+        public async Task<object> PostAsync(
+            [FromBody] MetricsRequestsApiModel requests,
+            string id)
+        {
+            var payload = requests?.ToServiceModel();
```

Comment:
We don't need to validate the received API model before processing it?

Reply:
No need for crawl.

Reply:
Nit: "API payload validation is not required as we're simply relaying the request."